lib/runtime/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+# SPDX-License-Identifier: Apache-2.0
+# SPDX-FileCopyrightText: 2019-2024 Second State INC
+
+wasmedge_add_library(wasmedgeRuntime
```

Comment:
We supposed that `wasmedgeRuntime` to be a header-only library.
Would you consider this?